Reports/reportModels.js

```diff
@@ -0,0 +1,26 @@
+var mongoose = require('mongoose');
```

Comment:
@jmtaber129 I think what @the-pat is looking for is isolated MongoDB and thus mongoose code out so if we decide to switch DB providers it will be easy.